glacier_analysis_grouped.ipynb

```diff
@@ -34,14 +34,14 @@
     "import pandas as pd\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    se_asia = gpd.read_file('/home/emmamarshall/Desktop/data/rgi/south_asia_east_15/15_rgi60_SouthAsiaEast.shp')
</pre><p>consider including a parquet file here too (as was done in a previous notebook)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/e-marshall/itslive/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>